src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -446,6 +474,27 @@ private string StripFormatSpecifier(string exp, out string formatSpecifier)
             return exp;
         }
 
+        private async Task<string> GetDereferencedTypeString(string expr, int threadId, uint frameLevel)
```

Comment:
>string [](http://example.com/codeflow?start=27&length=6)

If this fails, do we want to either throw or return a `Task<(string /*expression*/, string /*error*/)>` instead?

Reply:
I guess it doesn't really matter, since that expression should fail anyway.

Reply:
I'm hoping to get the NatvisDiagnostic work in to help with logging errors.